modules/components/src/DataTable/TableToolbar/TableToolbar.js

```diff
@@ -16,6 +17,7 @@ const TableToolbar = ({
   onColumnsChange,
   onSQONChange,
   onFilterChange = () => {},
+  donFilterChange = debounce(onFilterChange, 300),
```

Comment:
This is going to recall debounce on every render, might be better to use recomposes [withPropsOnChange](https://github.com/acdlite/recompose/blob/master/docs/API.md#withpropsonchange) instead